docs/src/main/tut/docs/non-automatic-derivation.md

```diff
@@ -76,7 +76,8 @@ pureconfig.loadConfig[Person](conf)
 ### Manual
 
 When case class and sealed trait derivation is not needed or wanted, we can simply not import anything and define our
-reader using any of ways explained in [Supporting New Types](supporting-new-types.html):
+reader using any of ways explained in [Supporting New Types](supporting-new-types.html). The `forProductN` helper
```

Comment:
Since you are already using `forProductN` in Supporting New Types, you should probably remove "also".